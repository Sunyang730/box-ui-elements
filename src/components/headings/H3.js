// @flow
import * as React from 'react';
import classNames from 'classnames';

import './H3.scss';

type Props = {
    children: React.Node,
    className?: string,
};

const H3 = ({ children, className }: Props) => <h3 className={classNames('bdl-H3', className)}>{children}</h3>;

export default H3;
