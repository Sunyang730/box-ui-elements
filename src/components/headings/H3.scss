@import '../../styles/variables';

h3.bdl-H3 {
    @include font-family;

    color: $bdl-gray;
    font-size: 16px;
    font-weight: normal;
    letter-spacing: .3px;
    line-height: 24px;
}
