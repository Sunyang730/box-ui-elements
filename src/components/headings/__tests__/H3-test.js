// @flow
import * as React from 'react';

import H3 from '../H3';

describe('components/headings/H3', () => {
    const getWrapper = (props = {}) => shallow(<H3 {...props}>foo</H3>);

    test('should render H3', () => {
        const wrapper = getWrapper({ className: 'moo' });

        expect(wrapper).toMatchSnapshot();
    });
});
