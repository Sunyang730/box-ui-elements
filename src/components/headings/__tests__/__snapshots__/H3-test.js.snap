// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/headings/H3 should render H3 1`] = `
<h3
  className="bdl-H3 moo"
>
  foo
</h3>
`;
